db/schema.rb

```diff
@@ -11,23 +11,13 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-<<<<<<< HEAD
-ActiveRecord::Schema.define(version: 20161206021535) do
-=======
 ActiveRecord::Schema.define(version: 20161205164755) do
->>>>>>> master
+
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-<<<<<<< HEAD
-  create_table "games", force: true do |t|
```

Comment:
haha you're 99% of the way there already! the rest of your code is really clean, just making sure you didn't delete all of your hard work by accident 😝